test_video.py

```diff
@@ -0,0 +1,208 @@
+# Basic options
+parser.add_argument('--content_video', type=str,
+                    help='File path to the content video')
+parser.add_argument('--style_path', type=str,
+                    help='File path to the style video or single image')
+parser.add_argument('--vgg', type=str, default='models/vgg_normalised.pth')
+parser.add_argument('--decoder', type=str, default='models/decoder.pth')
+
+# Additional options
+parser.add_argument('--content_size', type=int, default=512,
+                    help='New (minimum) size for the content image, \
+                    keeping the original size if set to 0')
+parser.add_argument('--style_size', type=int, default=512,
+                    help='New (minimum) size for the style image, \
+                    keeping the original size if set to 0')
+parser.add_argument('--crop', action='store_true',
+                    help='do center crop to create squared image')
+parser.add_argument('--save_ext', default='.mp4',
+                    help='The extension name of the output video')
+parser.add_argument('--output', type=str, default='output',
+                    help='Directory to save the output image(s)')
+
+# Advanced options
+parser.add_argument('--preserve_color', action='store_true',
+                    help='If specified, preserve color of the content image')
+parser.add_argument('--alpha', type=float, default=1.0,
+                    help='The weight that controls the degree of \
+                             stylization. Should be between 0 and 1')
+parser.add_argument(
+    '--style_interpolation_weights', type=str, default='',
+    help='The weight for blending the style of multiple style images')
+
+args = parser.parse_args()
+
+do_interpolation = False
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+output_dir = Path(args.output)
+output_dir.mkdir(exist_ok=True, parents=True)
+
+# Either --content or --contentDir should be given.
+assert (args.content_video)
+if args.content_video:
+    content_path = Path(args.content_video)
+
+# Either --style or --styleDir should be given.
+assert (args.style_path)
+if args.style_path:
+    style_path = Path(args.style_path)
+
+decoder = net.decoder
+vgg = net.vgg
+
+decoder.eval()
+vgg.eval()
+
+decoder.load_state_dict(torch.load(args.decoder))
+vgg.load_state_dict(torch.load(args.vgg))
+vgg = nn.Sequential(*list(vgg.children())[:31])
+
+vgg.to(device)
+decoder.to(device)
+
+content_tf = test_transform(args.content_size, args.crop)
+style_tf = test_transform(args.style_size, args.crop)
+        
+#get video fps&video size
+content_video = cv2.VideoCapture(args.content_video)
+fps = int(content_video.get(cv2.CAP_PROP_FPS))
+content_video_length = int(content_video.get(cv2.CAP_PROP_FRAME_COUNT))
+output_width = int(content_video.get(cv2.CAP_PROP_FRAME_WIDTH))
+output_height = int(content_video.get(cv2.CAP_PROP_FRAME_HEIGHT))
+
+assert fps!=0, 'Fps is zero, Please enter proper video path'

```

Comment:
Could you consider using the code formatter for Python such as [autopep8](https://pypi.org/project/autopep8/0.8.2/) for maintaining consistency with my codes?
I think whitespace should be placed before and after arithmetic operators.
(And this applies to the whole code)